contracts/src/ERC20Destination.sol

```diff
@@ -37,28 +38,18 @@ contract ERC20Destination is IERC20Bridge, TeleporterTokenDestination, ERC20 {
 
     /**
      * @notice Initializes this destination token bridge instance to receive
-     * tokens from the specified source chain and token bridge instance.
+     * tokens from the specified source chain and token bridge instance, and represents the
+     * received tokens with ERC20 tokens on this chain.
+     * @param settings Construction settings for this destination token bridge instance.
+     * @param tokenName The name of the ERC20 token.
```

Comment:
Looks like we're missing `tokenSymbol` here. Also may have meant "Constructor settings" rather than "Construction settings", but I'm not sure.